src/index.html

```diff
@@ -4,9 +4,384 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com/"
+    crossorigin="true"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com/" >
+    <link rel="stylesheet" href="styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          class="header__logo" 
+          alt="images_logo" 
+          src="images/logo.png"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-el">
+            <a class="nav__list-link is-active" href="#Apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Samsung" class="nav__list-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Smartphones" class="nav__list-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-el" data-qa="nav-hover">
+            <a href="#Laptop&Computers" class="nav__list-link">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Gadget" class="nav__list-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Tablets" class="nav__list-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Photo" class="nav__list-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list-el">
+            <a href="#Video" class="nav__list-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div data-qa="card" class="card">
```

Comment:
You should follow the BEM 
```suggestion
    <main class="cards">
      <div data-qa="cards_card" class="card">
```